JSONModel/JSONModel/JSONModel.m

```diff
@@ -721,6 +718,29 @@ -(void)__inspectProperties
                              );
 }
 
+-(void)__addCustomSetterForName:(NSString*) name
+                     withSuffix:(NSString*) suffix
+                            key:(NSString*) key
+                     toProperty:(JSONModelClassProperty*) p{
+    if (p.customSetters[key])
+        return;
+    
+    SEL setter = NSSelectorFromString([NSString stringWithFormat:@"set%@With%@:", name, suffix]);
```

Comment:
From my point of view, anyway, I should first check, does the selector exist, and only then create JSONModelCustomSetter. So I can only hide into the class the magic of wrapping into the NSValue, but